WordPress/UITestsFoundation/Screens/StatsScreen.swift

```diff
@@ -4,10 +4,17 @@ import XCTest
 public class StatsScreen: ScreenObject {
 
     public enum Mode: String {
+        case insights
         case months
         case years
     }
 
+    private enum Timeouts {
+        static let short: TimeInterval = 1
```

Comment:
I like the idea of having timeouts for several cases 👍 
However, is a one-second timeout really reliable and enough?

Reply:
I stole it from @mokagio without even sending him a note. =P

I just wanted to try it out, so it really wasn't a thoughtful decision on those numbers but I saw the `3 ` seconds all over the place so it felt like a `default` and enough time to load elements depending on additional API calls, so I looked at `1` second timeout for elements/screens that are not dependent on API calls, like pop-up validation for an empty required field or settings screen. However, as the tests are taking a life to run and this 2 seconds wouldn't make much difference, we may end up never using the `1` second and always sticking to the `default` _just to be safe_.